static/css/footer.less

```diff
@@ -189,133 +121,118 @@
         position: relative;
         margin-bottom: unit(45px / @base-font-size-px, em);
 
+        .footer_top-button {
+            display: block;
+            width: 90%;
+            max-width: 350px;
+            margin: -2em auto 2em;
+
+            .cf-icon-arrow-up {
+                padding-left: 1em;
+            }
+        }
+
+        .footer_nav-list {
+            margin-bottom: 0;
+        }
+
         .respond-to-min(768px, {
             padding-bottom: unit(30px / @base-font-size-px, em);
             margin-bottom: unit(30px / @base-font-size-px, em);
             border-bottom: 1px solid @gray-50;
-        });
 
-        .footer_share-icon-list {
-            .respond-to-min(768px, {
-                position: absolute;
-                // Aligns with the baseline of .footer_nav-list text.
-                top: -9px;
-                right: 0;
-            });
-        }
+            .footer_top-button {
+                display: none;
```

Comment:
Use `u-visually-hidden();` if this contains content that still needs to be accessible to screen readers. If not, then this is fine.